### ISSUE TITLE
No table of common values provided

### ISSUE THREAD

**Issue by haydenmc** [MicrosoftDocs/winrt-api#1454]
The table referenced by this article does not exist.

&gt; The table below lists common values for the interface type although many other values are possible.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b03b049e-731b-7623-32e7-7c9ded9ebecf
* Version Independent ID: c7ab838b-c5e9-80fc-ea2a-bbb89491b28f
* Content: [NetworkAdapter.IanaInterfaceType Property (Windows.Networking.Connectivity) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.networking.connectivity.networkadapter.ianainterfacetype#feedback)
* Content Source: [https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.networking.connectivity.networkadapter.yml&version=GBlive&_a=contents](https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.networking.connectivity.networkadapter.yml&version=GBlive&_a=contents)
* Product: **uwp**
* Technology: **winrt-reference**
* Git Login: **stevewhims**
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
Hi, @haydenmc, thanks for filing this issue. I looked at the history of the topic, and that sentence was there from the beginning with no table. So I deleted the sentence, and instead directed the reader to the Remarks section. I've also notified the product team in case they happen to have that table.

I'll close this issue once my changes have made their way to the live site.

Thanks again,
Steve
